src/middleWares/auth.js

```diff
@@ -0,0 +1,21 @@
+import jwt from "jsonwebtoken";
+
+const verifyToken = (req, res, next) => {
+  const authHeader = req.headers.authorization;
+
+  if (authHeader) {
+    const token = authHeader.split(" ")[1];
+
+    jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, (err, user) => {
+      if (err) {
+        return res.sendStatus(403);
+      }
+
+      req.user = user;
+      next();
+    });
+  } else {
+    res.sendStatus(401);
+  }
+};
+export default verifyToken;
```

Comment:
@BiGelio Does the verifyToken middleware check if the token is blacklisted? Because Roger will be blacklisting them when logging out. And the token may be valid but blacklisted.